.github/workflows/build.yml

```diff
@@ -184,6 +184,8 @@ jobs:
     steps:
       - name: Checkout source code
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
The error message may have been a red herring. Instead, the location of the coverage data may be different in Rust 1.45 as @valentinewallace pointed out: https://github.com/rust-bitcoin/rust-lightning/pull/774#issuecomment-763250623